docs/providers/asgardeo.md

```diff
@@ -0,0 +1,52 @@
+---
+id: asgardeo
+title: Asgardeo
+---
+
+## Documentation
+
+https://wso2.com/asgardeo/docs/guides/authentication
+
+
+## Example
+
+### To allow specific Active Directory users access:
+
+- Log into https://console.asgardeo.io.
+- Next, go to "Application" in https://console.asgardeo.io/t/{your_tenant}/develop/applications .
```

Comment:
Isn't it better if we can guide this step through UI? Other than providing this URL. We can explain the flow in summary and link to the Asgardeo application creation doc for more information.

Reply:
yes that would be great. Will link the guide.